pkg/cluster/controllers/storage/tenant.go

```diff
@@ -247,19 +247,7 @@ func (m *MinIOController) getResourceRequirements(harborcluster *goharborv1.Harb
 		return &harborcluster.Spec.InClusterStorage.MinIOSpec.Resources
 	}
 
-	limits := map[corev1.ResourceName]resource.Quantity{
-		corev1.ResourceCPU:    resource.MustParse("250m"),
-		corev1.ResourceMemory: resource.MustParse("512Mi"),
-	}
-	requests := map[corev1.ResourceName]resource.Quantity{
-		corev1.ResourceCPU:    resource.MustParse("250m"),
-		corev1.ResourceMemory: resource.MustParse("512Mi"),
-	}
-
-	return &corev1.ResourceRequirements{
-		Limits:   limits,
-		Requests: requests,
-	}
+	return &corev1.ResourceRequirements{}
```

Comment:
Can we just return nil here?

Reply:
I don't sure this is right, but return `&corev1.ResourceRequirements{}` must be correct。

Reply:
looks like return `nil` works well,  and maybe it is a better choice, and to avoid memory escapes.